app/views/csv_files/_csv_file.html.erb

```diff
@@ -0,0 +1,18 @@
+<tr>
+	<td><%= csv_file.id %></td>
+	<td><%= csv_file.name %></td>
+	<td><%= csv_file.humanize_type %></td>
+	<td><%= csv_file.humanize_date %></td>
+
+  <td>
+  	<%= link_to 'Show', csv_file %>
+  </td>
+  <td>
+  	<%= link_to 'Destroy', csv_file, method: :delete, data: { confirm: 'Are you sure?' } %>
```

Comment:
Design nitpick: I remember a conversation with @rickleegit or someone else on not calling this function `Destroy`. Was another word settled upon?